tools/rosbash/scripts/rosrun

```diff
@@ -50,8 +50,8 @@ esac
 if [[ -n $CMAKE_PREFIX_PATH ]]; then
   IFS=$'\n'
   catkin_package_libexec_dirs=($(catkin_find --without-underlays --libexec --share "$1" 2> /dev/null))
+  debug "Looking in catkin libexec dirs: $IFS$(catkin_find --without-underlays --libexec --share "$1" 2>&1)"
```

Comment:
Why should the `stderr` output be redirected to `stdout`?

Reply:
This ensures that we do not alter the known behavior of `rosrun` w/o `--debug`. As mentioned by you in an [earlier post](https://github.com/ros/ros/pull/182#discussion_r251801568), there might be installed packages which generate warnings, but which is totally OK. If we do not redirect `stderr` to `stdout` and run `rosrun` w/o `-d`, the user sees those warnings, because they are not fetched by the empty `debug` function. However, the new behaviour of `rosrun` w/ `-d` is now, that we see the warnings and errors produced by `catkin_make`, which gives the user more insights why his execution fails.